### PR TITLE
Fix action bar bugs

### LINKED ISSUE #542: Actionbar bugs
### In the video sheet: 

- gap on desktop view: 
image
- action bar not expanding to full width on smaller devices
image
- action bar covering the part of video form 
image

### In the create/edit channel view
- When the draft bar is visible, it will cover the Action bar. To reproduce - go to channels, try to edit your channel, click upload video and minimize the video sheet
image

I think that's all.
If you find anything, maybe comment here and I'll update the list above.

### PR DESCRIPTION
**Pull request by drillprop**
Fix #542

### CODE REVIEW

**Comment by mikkio-j** [comment]
It has slightly different transition right now. I believe channel's action bar needs some delay on entering when video sheet is sliding down. Without it those two bars are overlapped for a split second

File: src/shared/components/ActionBar/ActionBarTransaction.style.ts [hunk 1]
```diff
@@ -10,8 +10,9 @@ type ActionBarTransactionWrapperProps = {
 export const ActionBarTransactionWrapper = styled.div<ActionBarTransactionWrapperProps>`
   position: fixed;
   bottom: 0;
-  left: ${({ fullWidth }) => (fullWidth ? 0 : 'var(--sidenav-collapsed-width);')};
-  width: ${({ fullWidth }) => (fullWidth ? '100%' : 'calc(100% - var(--sidenav-collapsed-width))')};
+  left: ${({ fullWidth }) => (fullWidth ? 0 : 'var(--sidenav-collapsed-width)')};
+  width: ${({ fullWidth }) => (fullWidth ? '100%' : ' calc(100% - var(--sidenav-collapsed-width))')};
```
Comment (kdembler) [inline src/shared/components/ActionBar/ActionBarTransaction.style.ts]: This can be just `right: 0;`

**Review by eldiegod — APPROVED** [review]
Looks like all the issues were fixed 👌

**Review by mikkio-j — APPROVED** [review]
🎉

**Pull request merged**